abs.yaml

```diff
@@ -1,2 +0,0 @@
-channels:
-  - c3i_test2
```

Comment:
could you please use in future a different channel name for such things? The channel c3i_test2 is used by QA on miniconda/anaconda test runs.  So having there random files in that channel might lead to issues on their side.
I have for such things a private channel, which I clean up time by time